specification/purview/data-plane/Azure.Analytics.Purview.DataMap/preview/2023-10-01-preview/purviewdatamap.json

```diff
@@ -1551,6 +1630,181 @@
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/entity/operations": {
+      "get": {
+        "tags": [
+          "Entity"
+        ],
+        "description": "List entity asynchronous operations.",
+        "operationId": "Entity_ListOperationStatus",
+        "x-ms-examples": {
+          "Entity_ListOperationStatus": {
+            "$ref": "./examples/Entity_ListOperationStatus.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "dayRange",
+            "in": "query",
+            "description": "Return the import operations within the day range. The value will be 7 by default.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "type",
+            "in": "query",
+            "description": "The job type for the asynchronous operation",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/purviewAPIVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Return the status of operations.",
+            "schema": {
+              "type": "array",
+              "description": "An array of operations.",
+              "items": {
+                "$ref": "#/definitions/EntityAsyncOperation"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/entity/operations/{operationGuid}": {
+      "get": {
+        "tags": [
+          "Entity"
+        ],
+        "description": "Get the status of entity asynchronous operation",
+        "operationId": "Entity_GetOperationStatus",
+        "x-ms-examples": {
+          "Entity_GetOperationStatus": {
+            "$ref": "./examples/Entity_GetOperationStatusByOperationGuid.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/operationGuid"
+          },
+          {
+            "$ref": "#/parameters/purviewAPIVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Return the status of an entity async operation.",
+            "schema": {
+              "$ref": "#/definitions/EntityAsyncOperation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/entity/operations/{operationGuid}/cancel": {
```

Comment:
Should be `/entity/operations/{operationGuid}:cancel`. See our POST action guidelines: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#performing-an-action